docs/PlatformSpecificInformation.md

```diff
@@ -55,6 +55,41 @@ var styles = StyleSheet.create({
 
 `Platform.OS` will be `ios` when running in iOS and `android` when running in an Android device or simulator.
 
+There is also a `Platform.select` method available, that given an object containing Platform.OS as keys,
+returns the value for the platform you are currently running on.
+
+```javascript
+var { Platform } = React;
```

Comment:
Would you mind clarifying? (Is the PR update allowed when shipping in progress?)